src/pages/api/admin/statistics/getTableData.ts

```diff
@@ -30,17 +45,33 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
         user: 'admin',
       });
       await questdbClient.connect();
-      const sessionsData = await questdbClient.query(`SELECT * FROM '${DB_TABLE_NAMES.GLOBAL_SESSIONS}'`);
-      const workspacesData = await questdbClient.query(`SELECT * FROM '${DB_TABLE_NAMES.GLOBAL_WORKSPACES}'`);
-      const globalUsageData = await questdbClient.query(`SELECT * FROM '${DB_TABLE_NAMES.GLOBAL_USAGE}'`);
+      const globalWorkspaceList = await questdbClient.query(
+        `SELECT * FROM '${DB_TABLE_NAMES.GLOBAL_WORKSPACE_LIST}' WHERE isDeleted = false`
+      );
+      const workspaceListFromK8s = await k8s.getWorkspaceList();
```

Comment:
K8s API is the single source of truth for workspaces and it already comes with user information for each workspace.